gr-utils/modtool/templates/gr-newmod/python/CMakeLists.txt

```diff
@@ -32,13 +32,10 @@ include(GrTest)
 
 set(GR_TEST_TARGET_DEPS gnuradio-howto)
 
-# Create a module directory that tests can import. It includes everything
-# from `python/` and the built bindings shared lib.
+# Copy in all python source to a build directory python package for QA
+file(MAKE_DIRECTORY ${CMAKE_BINARY_DIR}/test_modules/howto)
 add_custom_target(
   copy_module_for_tests ALL
-  COMMAND ${CMAKE_COMMAND} -E copy_directory ${CMAKE_CURRENT_SOURCE_DIR}
+  COMMAND ${CMAKE_COMMAND} -E copy "${CMAKE_CURRENT_SOURCE_DIR}/*.py"
```

Comment:
Would anyone ever have a Python source file pull in some non-Python data, say a `coefficients.txt` file in the same directory? I don't know if that ever happens. If it did, just copying `.py` files over would break that. So it may be safer to just grab everything, I don't think that has side-effects.

Reply:
I did this thinking that custom or extra files would be handled with custom Cmake, and to leave that to the OOT developer. I'd be fine copying over everything though as well just to simplify things. Anyone else have an opinion on this?

Reply:
Talked it over a bit and decided copying over everything isn't dangerous and could be useful as you suggested. Just force pushed an update.